compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/SystemTheme.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui
+
+import androidx.compose.runtime.staticCompositionLocalOf
+import org.jetbrains.skiko.SystemTheme as SkikoSystemTheme
+import org.jetbrains.skiko.currentSystemTheme
+
+enum class SystemTheme {
```

Comment:
> Exposes skiko SystemTheme

Adding the enum inside `androidx.*` is actually better than exposing skikos one as mentioned in description 👍 

I know we already have public APIs with `org.jetbrains.skiko.*` types, but it's [not the right thing to do](https://github.com/JetBrains/compose-multiplatform/issues/3152). We're thinking how to deal with existing API (JFYI)

Reply:
Good suggestion.

@alexzhirkevich 
As I see - it is already changed? Maybe we can close this conversaion.